src/screens/Customers/components/CustomerCard.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import { Card, CardActions, Typography, IconButton } from '@material-ui/core';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+import { Link } from 'react-router-dom';
+import { setCurrentCustomerIdInRedux } from '../../../lib/redux/customerData';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    cardContent: {
+      flexGrow: 1,
+      display: 'flex',
+      alignItems: 'center',
+      paddingLeft: '17px',
+      color: theme.palette.text.primary,
+    },
+    totalText: {
+      color: theme.palette.text.secondary,
+    },
+    numberText: {
+      color: theme.palette.error.main,
+      marginLeft: '5px',
+    },
+    divSpacing: {
+      paddingLeft: '15px',
+    },
+    singleCard: {
+      width: '100%',
+      height: '93px',
+      border: '1px solid ' + theme.palette.text.disabled,
+      borderRadius: '8px',
+      textAlign: 'left',
+      display: 'flex',
+      flexDirection: 'row',
+      marginBottom: '10px',
+    },
+    active: {
+      height: '7px',
+      width: '7px',
+      borderRadius: '50%',
+      backgroundColor: theme.palette.secondary.light,
+    },
+    notActive: {
+      height: '7px',
+      width: '7px',
+      borderRadius: '50%',
+      backgroundColor: theme.palette.text.disabled,
+    },
+    arrowSpacing: {
+      padding: '0px',
+    },
+    indicator: {
+      display: 'flex',
+      alignItems: 'center',
+      marginRight: '10px',
+    },
+    icon: {
+      borderRadius: '8px',
+      color: 'white',
+      padding: '5px',
+      backgroundColor: theme.palette.primary.main,
+      fontSize: '30px',
+    },
+  });
+
+interface CardProps {
+  name: string;
+  customerId: string;
+  meterNumber: number;
+  amount: string;
+  active: boolean;
```

Comment:
I think it might make more sense to just pass the entire customer here so the props are cleaner? Kinda indifferent though, your call!